docs/sources/enterprise/kms-integration/using-aws-kms-to-encrypt-database-secrets.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "AMS KMS"
```

Comment:
This is adding AWS KMS content, not refactoring Hashicorp Vault content. Not a huge deal, but should this be a different PR?

Reply:
I changed the name of the PR. :)

Reply:
Changed the name of the PR again. :)